test_scripts/API/DeleteCommand/ATF_DeleteCommand_VR_DeleteCommand_WARNINGS.lua

```diff
@@ -0,0 +1,191 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one HMI-portions
+--
+-- Description:
+-- test is intended to check that SDL sends WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+-- in this test case when VR.DeleteCommand gets WARNINGS is checked
+-- 1. Used preconditions: App is activated and registered SUCESSFULLY
+-- 2. Performed steps: 
+-- MOB -> SDL: sends DeleteCommand
+-- SDL -> HMI: resends VR.DeleteCommand with imageType of cmdIcon which is not supported by HMI and UI.DeleteCommand with all valid params
```

Comment:
@vvvakulenko, can you clarify "with imageType of cmdIcon which is not supported by HMI". How will you make SDL to send such not supported type?